### PR TITLE
Hw6 Selenium - grid

### PR DESCRIPTION
**Pull request by solomencev**
image
image

### CODE REVIEW

File: src/test/java/com/epam/tc/hw6/DriverFabric.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package com.epam.tc.hw6;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.util.Locale;
+import lombok.NonNull;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+public class DriverFabric  {
```
Comment (Dr-Java) [inline src/test/java/com/epam/tc/hw6/DriverFabric.java]: Not Fabric, but Factory :)

File: src/test/java/com/epam/tc/hw6/DriverFabric.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    public static WebDriver getWebDriver(final boolean isLocal, final String hub, @NonNull final String browser) {
```
Comment (Dr-Java) [inline src/test/java/com/epam/tc/hw6/DriverFabric.java]: Parameters final boolean isLocal and final String hub are never used...

File: src/test/java/com/epam/tc/hw6/DriverFabric.java (hunk 1 above)
Comment (solomencev) [reply src/test/java/com/epam/tc/hw6/DriverFabric.java]: I used the suggestions of the lecturer from the previous course which you shared with me

File: src/test/java/com/epam/tc/hw6/DriverFabric.java (hunk 2 above)
Comment (solomencev) [reply src/test/java/com/epam/tc/hw6/DriverFabric.java]: I used the suggestions of the lecturer from the previous course which you shared with me. The framework has major changes..

**Review by Dr-Java — APPROVED** [review]

**Pull request merged**